packit_service/worker/handlers/distgit.py

```diff
@@ -757,7 +758,9 @@ def run(self) -> TaskResults:
             return super().run()
 
 
+@run_for_comment(command="ci")
```

Comment:
I would go with more granular commands, like in the service itself, i.e. `scratch-build` for example. And to preserve what current Fedora CI has, we could have `ci` that would trigger all the jobs, WDYT?

Reply:
I was thinking about `scratch-build` but then I thought it would be better to have the CI commands more distinguishable to users. Since we want to eventually allow users to retrigger individual checks/parts of the CI pipeline, how about something like `/packit ci[scratch-build]` or `/packit ci scratch-build`?

Reply:
That's a good point. I remember originally discussing maybe we will use a different command prefix for Fedora CI, so we could end up with `/packit-ci scratch-build`, WDYT? Maybe worth discussing with the rest of the team.

Reply:
If we change the prefix then `scratch-build` makes sense. What about the instances though? Let me change the command to `scratch-build` for now and we can discuss this on arch next week and change it.